accounts/keystore/keystore.go

```diff
@@ -446,7 +447,7 @@ func (ks *KeyStore) Import(keyJSON []byte, passphrase, newPassphrase string) (ac
 	ks.importMu.Lock()
 	defer ks.importMu.Unlock()
 	if ks.cache.hasAddress(key.Address) {
-		return accounts.Account{}, errors.New("account already exists")
+		return accounts.Account{}, ErrAlreadyExists
```

Comment:
I think it would make sense to return the account information here instead
of the zero value. In cmd/faucet, the assumption is that the account returned
by Import can be used for unlocking. This won't work with the zero value.

Reply:
That makes sense to me. Do you want to bundle that change into this PR, or keep the PRs atomic and I can make a follow up proposal implementing this?